src/lib/libwasmfs_fetch.js

```diff
@@ -49,11 +49,14 @@ addToLibrary({
             fileInfo.headers.has('Content-Length') &&
             fileInfo.headers.get('Accept-Ranges') == 'bytes' &&
             (parseInt(fileInfo.headers.get('Content-Length'), 10) > chunkSize*2)) {
+          var size = parseInt(fileInfo.headers.get('Content-Length'), 10);
           wasmFS$JSMemoryRanges[file] = {
-            size: parseInt(fileInfo.headers.get('Content-Length'), 10),
+            size,
             chunks: [],
             chunkSize: chunkSize
           };
+          firstChunk = 0;
```

Comment:
Please add a comment here, as why firstChunk should be 0 is not obvious to me.

Reply:
You're right, that's an error.